packages/edit-site/src/components/site-hub/index.js

```diff
@@ -101,8 +99,7 @@ const SiteHub = memo(
 							className="edit-site-site-hub__actions"
 						>
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit and check for search icon on the site editor mode.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="site-hub-command-before" src="https://github.com/user-attachments/assets/1426ce35-c2b7-4054-8197-2ede37e67cfd">
 | <img width="419" alt="site-hub-command-after" src="https://github.com/user-attachments/assets/983ba65c-519b-44c7-a20c-266c365b53f5"> |